Preparation/MyStack/MyStackInter.cs

```diff
@@ -9,6 +9,8 @@ public class MyStack<T> : IEnumerable<T>
         private int _count = -1;
         private T[] _underliyngArray = new T[5];
 
+        public event Action<int, bool> ItemAddOrDelete;
```

Comment:
1) It would be better to divide this into 2 events - ItemAdded & ItemDeleted.
2) This implementation will not support work with some custom class